export const handleError = (err) => console.log('Operation failed');